### DISCUSSION TITLE
post/authentik-tutorial-2-setup-reverse-proxy/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#17] [Blog Post Discussion]
# post/authentik-tutorial-2-setup-reverse-proxy/

Homepage and blog for Zhenghao Wu (ECWU), sharing thoughts on technology, everyday experiences, and photography. Engaging with the community through knowledge sharing.

https://ecwuuuuu.com/post/authentik-tutorial-2-setup-reverse-proxy/

**Comment by WallBreakerNO4** [comment]
如果是像我这样的懒人，直接用Caddy一行反代9000端口也是可以的 (●'◡'●)
```Caddyfile
example.com {
    reverse_proxy http://[your authentik server]:9000
}
```
